airbyte-integrations/connectors/source-it-glue/source_it_glue/source.py

```diff
@@ -0,0 +1,452 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+from urllib.parse import parse_qs, urlparse
+
+import requests
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+
+
+# Basic full refresh stream
+class ItGlueStream(HttpStream, ABC):
+
+    # FIRST_PAGE = 1
+    LIMIT = 1000
+
+    # TODO: Fill in the url base. Required.
```

Comment:
remove this

Reply:
I've limited the page to 1000 because, there are some end point like user_metrics which generates 120K records and It was the best move to make the connector run faster.